test/parallel/test-eventtarget-whatwg-signal.js

```diff
@@ -157,3 +158,11 @@ const {
   }, { once: true });
   et.dispatchEvent(new Event('foo'));
 }
+{
+  const et = new EventTarget();
+  [1, {}, null, false, 'hi'].forEach((signal) => {
```

Comment:
Let's add a few more of these.
```suggestion
  [1, 1n, {}, [], null, true, 'hi', Symbol(), () => {}].forEach((signal) => {
```